packages/compass-components/src/hooks/use-hotkeys.ts

```diff
@@ -0,0 +1,38 @@
+import { useMemo } from 'react';
+import { useHotkeys as useGlobalHotkeys } from 'react-hotkeys-hook';
+
+const isMac = () => navigator.userAgent.indexOf('Mac') !== -1;
+
+/**
+ *
+ * Modifier | macOS       | Windows/Linux
+ * -------- | ----------- | -------------
+ * ctrl     | control     | ctrl
+ * shift    | shift       | shift
+ * alt      | option      | alt
+ * meta     | command     | ctrl
+ *
+ */
+export const useHotkeys = (...args: Parameters<typeof useGlobalHotkeys>) => {
+  const [key] = args;
+  const ref = useGlobalHotkeys(...args);
```

Comment:
may be good to leave a comment that `meta` is already interpreted as Ctrl or Cmd by `useGlobalHotkeys`, I may be wrong on this, but it could be a bit unexpected, given that the meaning of `metaKey` is (or was) confusing even between browsers